powersimdata/input/change_table.py

```diff
@@ -646,6 +673,11 @@ def add_plant(self, info):
 
         :param list info: each entry is a dictionary. The dictionary gathers
             the information needed to create a new generator.
+            Required keys: "bus_id", "Pmax", "type".
+            Optional keys: "c0", "c1", "c2", "Pmin".
+            "c0", "c1", and "c2" are the coefficients for the cost curve, representing
+            the fixed cost ($/hour), linear cost ($/MWh), and quadratic cost ($/MW^2*h).
```

Comment:
:math:\`\rm{MW}^2 \rm{h}\`

Reply:
Do you want formatting on the quadratic term only, or all of them?

Reply:
Will it change for the other ones? the exponent in the quadratic is the on;y thing that will be in math mode

Reply:
First time see this representation. This is the classic way to represent the unit of the quadratic term?

Reply:
@rouille: Ah, I get it now.

Reply:
:math: in ReST lets you right some LaTeX math

Reply:
@BainanXia I'm not sure. I think removing the `*` per @rouille's suggestion will make it clear. The units line up properly.

Reply:
Done.